wagtail/admin/views/generic/mixins.py

```diff
@@ -637,10 +637,13 @@ def get_lock_context(self):
                     _("Cancel scheduled publish"),
                 )
 
-            if self.locked_for_user:
-                messages.error(self.request, lock_message, extra_tags="lock")
-            else:
+            if (
+                not isinstance(self.lock, ScheduledForPublishLock)
+                and self.locked_for_user
+            ):
```

Comment:
I think i’d have found this a tad more readable the other way around `if self.locked_for_user and not isinstance(self.lock, ScheduledForPublishLock)`.

Reply:
I did too, but unfortunately Black will reformat it to read like this:

```py
            if self.locked_for_user and not isinstance(
                self.lock, ScheduledForPublishLock
            ):
```

which to me is less readable 😞

I suppose I could've added parentheses around the `not isinstance`, which would make Black format it as:

```py
            if self.locked_for_user and (
                not isinstance(self.lock, ScheduledForPublishLock)
            ):
```

which looks somewhat better. But oh well!

Reply:
oh well indeed :)